common/com/twitter/intellij/pants/util/PantsConstants.java

```diff
@@ -15,6 +15,7 @@
 
   @NotNull
   public static final ProjectSystemId SYSTEM_ID = new ProjectSystemId(PANTS);
+  public static final ProjectSystemId BSP_SYSTEM_ID = new ProjectSystemId("BSP");
```

Comment:
Perhaps it is safer to define it as "ProjectSystemId.findById("BSP")". I am not sure if the identity of project id is compared by the id value (string) or object reference. The actual object for project system id is created in scala plugin.